src/main/java/com/GeorgesServer/app/Router.java

```diff
@@ -0,0 +1,28 @@
+package com.GeorgesServer.app;
+
+import com.GeorgesServer.app.com.GeorgesServer.handler.IHandler;
+
+public class Router {
+
+    private IHandler defaultHandler;
+    private IHandler formHandler;
+    private IHandler handler;
+
+    public Router(IHandler defaultHandler, IHandler formHandler) {
+        this.defaultHandler = defaultHandler;
+        this.formHandler = formHandler;
+    }
+
+    public IHandler route(String method, String url) {
```

Comment:
This is fine for now, but start thinking about how you can make this more extensible.